airdrop/infrastructure/repositories/airdrop_window_repository.py

```diff
@@ -5,12 +5,12 @@
 
 class AirdropWindowRepository(BaseRepository):
 
-    def get_airdrop_window_by_id(self, airdrop_window_id):
-        return (
-            self.session.query(AirdropWindow)
-            .filter(AirdropWindow.id == airdrop_window_id)
-            .first()
-        )
+    def get_airdrop_window_by_id(self, airdrop_window_id, airdrop_id=None):
+        query = self.session.query(AirdropWindow).filter(AirdropWindow.id == airdrop_window_id)
+        if airdrop_id:
```

Comment:
What are we doing here?

Reply:
Normally airdrop window id is enough to get the airdrop window details but it's better to check against airdrop id as well.
Airdrop window id is unique across all the airdrop but it's the logic at db level or infrastructure level.

Reply:
Where is airdrop id being used?

Reply:
Airdrop id is used as an additional check.